otter/grade/utils.py

```diff
@@ -151,3 +152,15 @@ def merge_scores_to_df(scores: List[GradingResults]) -> pd.DataFrame:
         df_scores = pd.DataFrame(scores_dict, index=[0])
         full_df.append(df_scores)
     return full_df
+
+
+def monitor_progress(queue: Queue, msg: str):
```

Comment:
Since Otter already has logging, why not hook into that (and add your own logs as needed)? You could create a [logging handler](https://docs.python.org/3/library/logging.html#handler-objects) that handles logs by passing them to your `Queue` and do something similar to [this](https://github.com/ucbds-infra/otter-grader/blob/master/otter/utils.py#L268) in the `loggers` class to send any logs to that handler when it's set up.

Reply:
I pushed a commit that uses the logger. Take a look. I was not quite sure how to approach registering my handler with the loggers. Open to suggestions.